crates/tuono/src/app.rs

```diff
@@ -218,8 +219,13 @@ impl App {
         match Config::get() {
             Ok(config) => self.config = Some(config),
             Err(error) => {
-                eprintln!("[CLI] Failed to read tuono.config.ts with the following error:");
-                eprintln!("{}", error);
+                match error.kind() {
+                    io::ErrorKind::NotFound => eprintln!("[CLI] Failed to read config. Please run `npm install` to generate automatically."),
+                    _ => {
+                        eprintln!("[CLI] Failed to read tuono.config.ts with the following error:");
+                        eprintln!("{}", error);
```

Comment:
Since this is not a `handled` error I'd consider making it logged with the `tracing` logger

```suggestion
                        error!("[CLI] Failed to read tuono.config.ts with the following error:");
                        error!("{}", error);
```